src/components/OSCALDrawerSelector.js

```diff
@@ -0,0 +1,56 @@
+import { Divider, Drawer, IconButton, styled, useTheme } from "@mui/material";
+import React from "react";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import ListItemText from "@mui/material/ListItemText";
+
+const DrawerHeader = styled("div")(({ theme }) => ({
+  display: "flex",
+  alignItems: "center",
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+  justifyContent: "flex-end",
+}));
+
+export default function OSCALDrawerSelector(props) {
+  const theme = useTheme();
+  const [open, setOpen] = React.useState(false);
```

Comment:
We tend to use `is` for boolean variables: `isOpen`/`setIsOpen`

Reply:
Oh that does make more sense. This is really just me copying and pasting the example code on the [React documentation](https://mui.com/material-ui/react-drawer/#persistent-drawer) as a proof of concept but I will keep that in mind.